features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/app/store-device-type/themes/store/js/policy-add-new.js

```diff
@@ -0,0 +1,225 @@
+var deviceId, deviceType;
+$('select.select2').select2({
+    placeholder: 'Select..'
+});
+
+$('select.select2[multiple=multiple]').select2({
+    placeholder: 'Select..',
+    tags: true
+});
+
+var stepperRegistry = {},
+    hiddenOperation = '.wr-hidden-operations-content > div',
+    advanceOperation = '.wr-advance-operations';
```

Comment:
be consistent on using quotes for strings. better to use double quotes instead of single quotes.